### PR TITLE
Portfolio:Desktop version

### PR DESCRIPTION
**Pull request by Cheptoo21**
In this Project:

- I used media query to make my portfolio responsive.
- I added contact form.
- I implemented desktop version of the portfolio.
- I deployed my project using github pages.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+        <div class="heading">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h3>
+        </div>
+        <div class="form">
+            <form action="https://formspree.io/f/xayzredb" method="POST">
+                <ul>
+                    <li>
+                        <input name="user_Name" type="text" maxlength="30" placeholder="Full Name" >
```
Comment (nicupop729) [inline index.html]: As specified in the project requirements, it is stated that a user should not be unable to submit the form if any of the fields are left empty. However, in this particular case, I was able to submit the form without entering a name, which is something we need to address.

- [ ] Kindly ensure that inputting the name becomes a mandatory requirement for form submission.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+        </div>
+    </section>
+    <p class="email">cheptoopatience81@gmail.com</p>
+    <div class="background">
+        <img class="Background_img1" src="frames/form1.png" alt="Img 1">
+        <img class="Background_img2" src="frames/illustration-2-contact-form-desktop@2x.png" alt="Img 2">
+        <img class="Background_img3" src="frames/form.png" alt="Img 3">
+
+    </div>
```
Comment (nicupop729) [inline index.html]: The background images should always be introduced through the CSS file and never through the index.html file, to follow the best practices.
in this case, in the mobile view, it creates a big white space between the form and the social media icons.
https://www.loom.com/share/a56a46d2cf4d4ce49f09fae3f9593e29?sid=0d0b4c24-4477-4f7a-bf6e-a0d5d5b0e46c

- [ ] Kindly make sure to remove the white space between the form and the social media icons in the mobile view.
- [ ] Let's also ensure that we introduce the background images through the CSS file and not through the index.html file.

[Here is an article](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html#:~:text=In%20general%2C%20use%20the%20%3Cimg,for%20decorative%20or%20design%20purposes.) about how to introduce the background images through CSS file and not HTML file.

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hello @Cheptoo21  👋,

_This is an manual review._

Please review the inline comments provided for improvement suggestions to ensure the approval of these two milestones. 
Once you have made the necessary changes, kindly request another review.

Keep rocking 🚀!

**Comment by Cheptoo21** [comment]
Hello  ***@***.****
I hope this email finds you well.
Here is the link to my PR with the requested changes:
https://github.com/Cheptoo21/Project-Portfolio/pull/1.
Looking forward to the approval.
Thank you.

**Comment by bellom** [comment]
Hi @Cheptoo21,

### This is a manual review

# Approve :+1:
Cheers and Happy coding!:clap::clap::clap:

**Comment by bellom** [comment]
@Cheptoo21 - kindly go ahead and merge so that we can continue the process.

**Pull request merged**